Gemfile

```diff
@@ -7,8 +7,10 @@ source "https://rubygems.org"
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
+gem "bundler"
```

Comment:
I think requiring bundler in a Gemfile may be redundant. It would of course need to be listed as a dependency if we need to call internal bundler functions, but mostly people just use it for installing gems with `$ bundle` or `$ bundle install` on the command line.

Reply:
Fixed